governing-documents/project-annual-review.md

```diff
@@ -10,81 +10,89 @@ nav_order: 5
 
-Project maintainers are responsible for agreeing between them who will complete the annual review. One of the maintainers should create the review in GitHub under [hyperledger/toc/project-reports/](https://github.com/hyperledger/toc/tree/gh-pages/project-reports).
+LF Decentralized Trust staff will notify the project TSC and copy the TAC when the project review is due.
 
-* Raise a PR titled *[year] [Project Name] Annual Review* (e.g., 2024 Hyperledger Amazing Annual Review)
-* The PR should include a file called `./<year>/<year>-annual-<Project-Name>.md` (e.g., `2024/2024-annual-Hyperledger-Amazing.md`) using [the annual review template](../project-reports/0000-annual-review-template.md)
-    - Update the information at the top of the file:
-        - change the `title` line to `YYY Annual Review Project Name` (e.g., `title: 2024 Annual Review Hyperledger Amazing`)
-        - change the `parent` line to `YYYY` (e.g., `parent: 2024`)
-        - change the `grand_parent` to `Project Updates` (i.e., `grand_parent: Project Updates`)
-        - remove the `nav_exclude` line
-    - Text between `<mark></mark>` are instructions. Please remove when section has been completed.
-* Send an email to the [TAC mailing list](mailto:toc@lists.hyperledger.org) so that the community knows the PR is there and can comment on it.
+Project TSC members are responsible for agreeing between them who will complete the annual review. One of the TSC members should create the review in GitHub under [lf-decentralized-trust/toc/project-reports/](https://github.com/lf-decentralized-trust/toc/tree/gh-pages/project-reports).
+
+- Raise a PR titled *`[year] [Project Name] Annual Review`* (e.g., `2024 LFDT Amazing Annual Review`).
+- The PR should include a file called `./<year>/<year>-annual-<Project-Name>.md` (e.g., `2024/2024-annual-lfdt-amazing.md`) using [the annual review template](../project-reports/0000-annual-review-template.md).
+  - Update the information at the top of the file:
+    - change the `title` line to `YYY Annual Review Project Name` (e.g., `title: 2024 Annual Review Hyperledger Amazing`)
+    - change the `parent` line to `YYYY` (e.g., `parent: 2024`)
+    - change the `grand_parent` to `Project Updates` (i.e., `grand_parent: Project Updates`)
+    - remove the `nav_exclude` line
+  - Text between `<mark></mark>` are instructions. Please remove when section has been completed.
+- Send an email to the **LINK TBD** `TAC mailing list` so that the community knows the PR is there and can comment on it.
 
 ## Potential Outcomes
+
 The outcome of the annual review is:
 
-* A majority of the TAC members agree to continue the project at its current stage, or
-* A majority of the TAC members recommend that a project be moved to a new stage, including `End of Life`, or
-* If the TAC cannot come to a consensus, the project will remain at its current stage.
+- A majority of the TAC members agree to continue the project at its current status, or
```

Comment:
Q: Is the term `project's status` broader than `project's stage` or do they carry the same meaning? If `status` is broader, does it imply an inferred condition based on the project's report and current stage?

Reply:
As per the discussion at the TOC call on 20240829 — the term “status” and “stage” were used seemingly interchangeably in the document.  I didn’t think that was clear, so I checked the “lifecycle” document and it uses the term “status”, so I changed all “stage” references to “status”.